### PR TITLE
Adding Javascript setup info

### PR DESCRIPTION
**Pull request by manoj9788**

### CODE REVIEW

File: src/main/rst/03_webdriver.rst [hunk 1]
```diff
@@ -960,6 +1037,11 @@ popup, you can access the alert with the following:
 
     $driver->get_alert_text();
     $driver->accept_alert();
+    
+.. code-block:: javascript
+
+	var alert = new new driver.switchTo().alert();
```
Comment (ddavison) [inline src/main/rst/03_webdriver.rst]: a little redundancy :)

File: src/main/rst/03_webdriver.rst [hunk 2]
```diff
@@ -171,6 +171,16 @@ PHP bindings are provided by a third party, please refer to any of their documen
 install / get started. There are three known bindings at this time: `By Chibimagic <https://github.com/chibimagic/WebDriver-PHP/>`_
 `By Lukasz Kolczynski <http://code.google.com/p/php-webdriver-bindings/>`_ and `By the Facebook <https://github.com/facebook/php-webdriver>`_
 
+JavaScript
+~~~~
+If you are using node.js to develop your application then you are better off using the `Javascript bindings <https://www.npmjs.com/package/selenium-webdriver>`_ of the selenium project.
```
Comment (lukeis) [inline src/main/rst/03_webdriver.rst]: not sure i like saying "better off" can we change this to a more definitive statement?

"If you are using node.js to develper you application, the selenium project provides `Javascript bindings on npm`<...>_"

File: src/main/rst/03_webdriver.rst [hunk 3]
```diff
@@ -171,6 +171,16 @@ PHP bindings are provided by a third party, please refer to any of their documen
 install / get started. There are three known bindings at this time: `By Chibimagic <https://github.com/chibimagic/WebDriver-PHP/>`_
 `By Lukasz Kolczynski <http://code.google.com/p/php-webdriver-bindings/>`_ and `By the Facebook <https://github.com/facebook/php-webdriver>`_
 
+JavaScript
+~~~~
+If you are using node.js to develop your application then you are better off using the `Javascript bindings <https://www.npmjs.com/package/selenium-webdriver>`_ of the selenium project. 
+
+It can be easily installed via npm with
```
Comment (lukeis) [inline src/main/rst/03_webdriver.rst]: "It"? I would change this to "Install the Javascript bindings with npm:"

File: src/main/rst/03_webdriver.rst (hunk 2 above)
Comment (lukeis) [inline src/main/rst/03_webdriver.rst]: We should also link in this section to the API docs -

http://seleniumhq.github.io/selenium/docs/api/javascript/index.html

File: src/main/rst/03_webdriver.rst [hunk 4]
```diff
@@ -356,6 +374,13 @@ Example of how to find an element that looks like this:
     @cheeses = $driver->find_elements('cheese', 'class');
 
 
+.. code-block:: javascript
+	
+	findElements(By.className("cheese")).then(function(cheeses){
```
Comment (jleyba) [inline src/main/rst/03_webdriver.rst]: Less verbose if you use es6 arrow functions:

    driver.findElements(By.className("cheese"))
        .then(els => console.log(els.length));

File: src/main/rst/03_webdriver.rst (hunk 1 above)
Comment (jleyba) [inline src/main/rst/03_webdriver.rst]: also incorrect - it's not a constructor and can't be new'd.

    var alert = driver.switchTo().alert();

File: src/main/rst/03_webdriver.rst (hunk 2 above)
Comment (manoj9788) [inline src/main/rst/03_webdriver.rst]: Have corrected the typo in your suggestion will edit the changes accordingly.

File: src/main/rst/03_webdriver.rst (hunk 4 above)
Comment (manoj9788) [inline src/main/rst/03_webdriver.rst]: I had a dilemma on using that, glad you suggested the es6 one. Shall make the changes.

**Comment by manoj9788** [comment]
Is it good to merge now?

File: src/main/rst/03_webdriver.rst [hunk 5]
```diff
@@ -703,6 +756,13 @@ People often wish to retrieve the innerText value contained within an element.
 
     element = driver.find_element_by_id("element_id")
     element.text
+
+.. code-block:: javascript
+	
+	var element = driver.findElement(By.id('elementID'));
+	element.getText().then(function(text) {
```
Comment (jleyba) [inline src/main/rst/03_webdriver.rst]: var element = driver.findElement(By.id('elementID'));
    element.getText().then(text => console.log(`Text is ${text}`));

Comment (manoj9788) [inline src/main/rst/03_webdriver.rst]: Alright, will make sure its on es6 for all snippets

**Pull request merged**